README.md

```diff
@@ -370,7 +370,47 @@ c, _ := charge.Retrieve("ch_123", p)
 c.Customer.ID    // ID is still available
 c.Customer.Name  // Name is now also available (if it had a value)
 ```
+### How to use undocumented parameters and properties
 
+stripe-go is a typed library and it supports all public properties or parameters.
+
+Stripe sometimes has beta which introduces new properties or parameters that are not immediately public. The library does not support these properties or parameters until they are public but there is still an approach that allows you to use them.
```

Comment:
```suggestion
Stripe sometimes launches private beta features which introduce new properties or parameters that are not immediately public. These will not have typed accessors in the stripe-go library but can still be used. 
```

Reply:
This will be changing as we introduce support for betas.

Reply:
This makes sense and in light of those coming changes the revised language seems much more informative.